packages/embark/src/lib/modules/console/index.ts

```diff
@@ -102,8 +102,17 @@ class Console {
         let response = result;
         if (typeof result !== "string") {
           response = stringify(result, utils.jsonFunctionReplacer, 2);
+          this.logger.info(response);
+        } else {
+          // Avoid HTML injection in the Cockpit
+          this.logger.info(response);
+          response = response
+            .replace(/&/g, "&amp;")
+            .replace(/</g, "&lt;")
+            .replace(/>/g, "&gt;")
+            .replace(/\"/g, "&quot;")
+            .replace(/\'/g, "&#39;");
```

Comment:
There are npm packages that do that, but if we prefer our version of it, it's better to put it in a utils, because you use it at least twice.